src/pudl/metadata/constants.py

```diff
@@ -259,6 +259,34 @@
         "government",
         "federal",
     ],
+    "msha": [
+        "msha",
+        "mshamines",
+        "mine safety and health administration",
+        "mines",
+        "mining",
+        "coal",
+        "metal",
+    ],
+    "phmsa": [
```

Comment:
Might add "department of transportation" and "US DOT" since PHMSA is under them.